ext/enchant/config.m4

```diff
@@ -12,7 +12,8 @@ if test "$PHP_ENCHANT" != "no"; then
     PHP_EVAL_INCLINE([$ENCHANT2_CFLAGS])
     PHP_EVAL_LIBLINE([$ENCHANT2_LIBS], [ENCHANT_SHARED_LIBADD])
     AC_DEFINE([HAVE_ENCHANT_GET_VERSION], [1],
-      [enchant_get_version since 1.6.0])
+      [Define to 1 if Enchant library has the 'enchant_get_version' function
+      (available since 1.6.0).])
```

Comment:
seems enchant is the only extension having this kind of info.

Reply:
I think it is kind of useful info to know in which version the function was added without the need to browse the library changelog...